lua/nvim-treesitter/parsers.lua

```diff
@@ -761,6 +761,15 @@ list.llvm = {
   maintainers = { "@benwilliamgraham" },
 }
 
+list.http = {
+  install_info = {
+    url = "https://github.com/NTBBloodbath/tree-sitter-http",
+    branch = "main",
+    files = { "src/parser.c" },
```

Comment:
when doing inheritance, you will need to signalize that with `generate_requries_npm =true`